LICENSE

```diff
@@ -0,0 +1,15 @@
+DevOpsBootcamp
```

Comment:
Let's specifically call this ``OSUOSL DevOpsBootcamp`` so we don't get into trouble with other people using this term.